common/internal_model/src/main/scala/weco/catalogue/internal_model/index/WorksAnalysis.scala

```diff
@@ -8,6 +8,15 @@ object WorksAnalysis {
   val slashesCharFilter =
     MappingCharFilter("slashes_char_filter", mappings = Map("/" -> " __"))
 
+  // This analyzer "keeps" the hyphen, by removing it and treating hyphenated
+  // tokens as a single token.
```

Comment:
Does this mean that, for example, "e-mail" would be tokenised as "email"?

Reply:
exactly that! NB that only applies to (sub)fields which use the `hyphens_char_filter`, so we'll still be indexing a lot of "e"s and "mail"s, but we now have a better chance of retrieving docs which mention "email" over docs which mention "mail" when people search for "e-mail"